### ISSUE TITLE
Particle Editor

### ISSUE THREAD

**Issue by Technodweeb** [TheGameCreators/GameGuruRepo#3344]
I am using the Steam Version of both the Particle editor and GameGuru Max.
I am trying to  add new particles I created in PM or even just one that is in the Particle editor that is not in GGMax.

 I add a particle to the map, then go to add new particle, create a particle, or even just load a premade one, then I tell it to save it to GameGuruMax .  Nothing shows up in GGMax only the particles that exist in the GGMax already work for me. There is nothing in the user folder at all after saving.

I  checked in Steam and Verified The Integrity of the files, it said that two were bad and would be corrected. I restarted GGMax and Still had the issue.

**Comment by Technodweeb** [comment]
I am wondering if its looking at the wrong folder for the User Particles? Currently it says its saving in my main drive C:\users\loret\Documents\GameGuruApps\GameGuruMax\Files\particlesbank\user 
I can see all the Particles I created in that folder, they are just not showing up in GG Max..
I had the downloaded version installed before the steam version and wondering if that could have caused the issue. 
I did uninstall, the downloaded version,  before installing the steam version of both.
Thanks!

**Comment by Technodweeb** [comment]
Trouble Shooting continued;
I  decided  to move GGmax to the C Drive using Steams File Moving tool. That didn't solve the problem.
Particles I create were still not saving to the user folder.

I decided to remove GGmax completely, and start over. 
I  installed on the C: Drive. 
I opened GGmax and now when opening the particle editor to create a new particle, there is NO option to save it to GGMax just to save it   See image attached.

At this point I am baffled and will wait to hear from you.

thanks 
<img width="469" alt="PEPic" src="https://user-images.githubusercontent.com/61712610/175823636-1218d17a-b92f-43ea-806e-35f590f221e4.png">

**Comment by Granada1** [comment]
The team will get back to you soon , they don’t work weekends 👍

**Comment by Technodweeb** [comment]
Understood

**Comment by LeeBamberTGC** [comment]
@Technodweeb When the button to export to MAX does not appear, it means the Particle Editor does not know where it has been installed so does not know where to place it.  I imagine the repeated install and uninstall, moving drives, and many other adventures have left something in your system which is confusing the default approach to getting things up and running.  Can you use your D drive solution as a short-term measure for now, and we can add some debug info in a future build to find out why you are not seeing your particles export correctly?

**Comment by Technodweeb** [comment]
I removed and reinstalled to d: drive, and the particle editor still isn't giving me the option to export to GGMax. 
I will just wait for the Debug .

**Comment by LeeBamberTGC** [comment]
@Technodweeb I have added extra logging to the Steam EXP build and non-Steam latest release. Update to one of those and new debug info will be produced. If you can update then run, and send me the "Guru-MapEditor.log" file in "C:\Users\xxx\Documents\GameGuruApps\GameGuruMAX", that would be great.

**Comment by Technodweeb** [comment]
I tried to update but not seeing any options to do so.
I have the steam version it is still on the 06/24 build. I have steam set to Public Preview.  
I ran GGMax and then looked in the folder described and there is not Guru-MapEditir.log
But Good News... after reinstalling again, last night ,I am now getting the option to save the particles to Ggmax while in the particle editor... so that's working again.  I am still not seeing the particles I created in GGMax. Particle Editor is saving to C:\Users\xxx\Documents\GameGuruApps\GameGuruMAX\Files\particlesbank\user
Thanks for looking into this... Loretta

**Comment by LeeBamberTGC** [comment]
You will need to change your Steam build to the EXPERIMENTAL branch, use the Steam product properties and find the BETA tab. The code you need is: lettesterin456

You will find the LOG file in "C:\Users\xxx\Documents\GameGuruApps\GameGuruMAX" after launching.  If not, send me the file "...\steamapps\common\GameGuru MAX\setup.ini".

**Comment by LeeBamberTGC** [comment]
@Technodweeb With the "producelogfiles=1" field set, the .log file should be written. It may be hidden by your OS.  Any chance you can zip up the "C:\Users\xxx\Documents\GameGuruApps\GameGuruMAX" folder and email me at lee@thegamecreators.com. I can then see the entire contents of the writables area, thanks!

**Comment by Technodweeb** [comment]
That folder is 1.5GB in size and I can't send it via Email. I  instead uploaded it to one of my websites: https://lorettasview.com/GameGuruMAX.7z

**Comment by earthling45** [comment]
You could use google drive for those files and send Lee a link so only he has access to the files.
This is an invitation to everyone to pick those models which you have paid for or created i think.

**Comment by Technodweeb** [comment]
Thanks Earthling45 .. I wasn't thinking.. thanks for catching that. I will be sending you a link, Lee via email as you suggested.

**Comment by LeeBamberTGC** [comment]
@Technodweeb I have the files :)  Very good advice @earthling45  :)

**Comment by LeeBamberTGC** [comment]
@Technodweeb Have you changed the location of the writable area at all?  Maybe send a shot of this screen:

![image](https://user-images.githubusercontent.com/35840583/176284450-67d75d81-f17a-40dd-937d-ac8a054a021e.png)

**Comment by Technodweeb** [comment]
Yes! I forgot that!  I changed it to E:\GGMaxSaves\    and in that folder was the Guru_MapEditor.log  which I attached. 
[Guru-MapEditor.log](https://github.com/TheGameCreators/GameGuruRepo/files/9005909/Guru-MapEditor.log)

**Comment by LeeBamberTGC** [comment]
@Technodweeb Do you also find your particles you created?  If you changed the path to your E drive, it will not be in "C:\Users\xxx\Documents\GameGuruApps\GameGuruMAX\Files\particlesbank\user" but instead may be here:

E:\GGMaxSaves\Files\particlesbank\user

**Comment by Technodweeb** [comment]
No there are not saved particles in the user folder at E:\GGMaxSaves\Files\particlesbank\user

I can change it back to the default and see if they show up ... wanted to wait for your suggestion on that.

**Comment by LeeBamberTGC** [comment]
@Technodweeb It is worth a try, and if they work, then we have found a bug relating to moving the default writable location :)

**Comment by Technodweeb** [comment]
Ok will try that now...will reset that and reboot... back to you in a few...

**Comment by Technodweeb** [comment]
Good News! I put GGMAx back on the C: drive.  I then removed the particle editor through steam.  Then I did a search for particle_editor and found a folder under C:/user/loret/Appdata /Local/AGKApps/particle_editor  I deleted that folder and then reinstalled the particle editor and it all work Perfectly now...  Everything is installed in all the default folders and drives now. I did a test created a new particle and imported it in... works perfectly!
